lib/mixlib/install/product.rb

```diff
@@ -77,6 +79,10 @@ def default_value_for(prop)
           "/opt/#{package_name}"
         when :omnibus_project
           package_name
+        when :downloads_url
+          "https://downloads.chef.io/#{package_name}"
```

Comment:
This should actually be `"https://downloads.chef.io/#{product_key}"`. This will require we make `product_key` part of `Mixlib::Install::Product`'s `DSL_PROPERTIES`. We should be able to update `Mixlib::Install::Product`'s  initializer to take the key as the first argument:

```ruby
#
# The only DSL method of this class. It creates a Product with given
# `key` and stores it.
#
def product(key, &block)
  @product_map[key] = Product.new(key, &block)
end
```